### ISSUE TITLE
Possible to use own domain?

### ISSUE THREAD

**Issue by kyle95wm** [barronwaffles/dwc_network_server_emulator#102]
Okay so I talked about this a few times in IRC with Nagato, but unforuntatly I couldn't exactly do what I wanted to do.

Just recently I managed to set up a web-server that worked alongside my DWC server. The thing I wanted to do was make it possible to allow people to patch their game files to point to my server (very much the same way Wiimmfi works) the problem is, I don't exactly have a high-grade domain name system. I simply have http://beanjr.hopto.org which comes from http://noip.com

I tried adding address=/beanjr.hopto.org/99.238.209.85 to dnsmasq.conf and restarted everything but Dolphin simply crashes when trying to get to the connect screen. I have yet to try this on an actual Wii but I assume everything is a complete failure on my end. I've even went as far as editing the virtual host files in Apache 2 and enabling them.

Theres no excuse why this can't work like it does very much the same way 
address=/nintendowifi.net/99.238.209.85 does

NOTE: I've even tried adding address=/*.beanjr.hopto.org/99.238.209.85 to the config file but I have yet to succeed.

**Comment by kyle95wm** [comment]
anyone?

**Comment by mm201** [comment]
It's a problem with your server. Short of giving someone in here root access, I don't get how you expect us to help.

**Comment by kyle95wm** [comment]
ah thats true. I believe I've figured out the problem however there no hope in solving it. It seems that when I set my DNS to the IP of my server and used a patched game with the custom URL's they seem to work but outside the network since the domains aren't technically registered (but using a DDNS with no-ip) theres no such domain as contest.beanjr.hopto.org on the internet even though theres such a thing as beanjr.hopto.org on the internet (an nslookup should be able to tell you this)

Unfortunately I don't feel like paying a service like GoDaddy or anything of a sort just to have these extra domain names. If someone else is looking to do what I wanted to do and pay a company they could do it no problem.
